Update list of supported Products (#20161)
DataDog/documentation: content/en/account_management/billing/usage_attribution.md

- Summarizes usage at the end of each month.
- Surfaces the data both in the UI and as a TSV download.

> This feature does not support product usage that cannot be tagged during instrumentation (e.g., Incident Management Users, Parallel Testing Slots). 

Reviewer: ```suggestion
This feature does not support product usage that cannot be tagged during instrumentation. For example, Incident Management Users and Parallel Testing Slots. 
```
